### PR TITLE
OGM-1517 Add test persistence.xml without provider

### PR DESCRIPTION
**Pull request by fax4ever**
https://hibernate.atlassian.net/browse/OGM-1517

### CODE REVIEW

**Comment by fax4ever** [comment]
I've put `Preview` label, because we need to check better the test here.

File: core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.junit.rules.ExpectedException;
+
+/**
+ * Tests a store and load scenario, using the JPA interface,
+ * without defining a persistence provider to enable Hibernate OGM engine.
+ *
+ * @author Fabio Massimo Ercoli
+ */
+@TestForIssue(jiraKey = "OGM-1517")
+public class JPAStartWithoutProvider {
```
Comment (DavideD) [inline core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java]: Class name should end with `-Test`

File: core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	@Rule
+	public PackagingRule packaging = new PackagingRule( "persistencexml/ogm-noprovider.xml", Poem.class );
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void testJPAStartWithoutProvider() {
+		Map<String, Object> settings = new HashMap<>( TestHelper.getDefaultTestSettings() );
```
Comment (DavideD) [inline core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java]: Do you need this? Can't you simply pass the `Map<String, String>`?
```
factory = Persistence.createEntityManagerFactory( "ogm", TestHelper.getDefaultTestSettings() );
```

File: core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+			finally {
+				if ( manager != null ) {
+					manager.close();
+				}
+			}
+		}
+		finally {
+			if ( factory != null ) {
+				factory.close();
+			}
```
Comment (DavideD) [inline core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java]: I think that for this test I would prefer to close everything in a `@After` method. This way the test becomes much smaller and easier to follow. This is just a suggestion.

**Comment by fax4ever** [comment]
@DavideD thank you for the hints here. I'm going to close this PR.
In order to apply the strategy that we decided in the last meeting, described in the last comment of the issue itself.

**Pull request closed without merging**